tests/test_project_creation.py

```diff
@@ -83,7 +83,7 @@ def test_create_project_with_credentials(client, boto3_session):
     assert (
         created_proj
         and created_proj.date_created > time_before_run
-        and not created_proj.is_sensitive
+        and not created_proj.non_sensitive
```

Comment:
```suggestion
        and created_proj.non_sensitive
```
Not sure what it should be here (only looking at the diff), but shouldn't it be that way to maintain the meaning?

Reply:
Or maybe it's simply checking the defaults?